advanced/configuring-comments-callbacks.md

```diff
@@ -52,7 +52,11 @@ The done callback needs to take an object of the form:
 
 ```js
 {
-  conversationUid: string // the new conversation uid
+  conversationUid: string, // the new conversation uid
+  // optional error callback which will be executed in case the new conversation could not be created
+  onError: function (err) { ... },
+  // optional success callback which will be executed when the new conversation is successfully created
```

Comment:
```suggestion
  // Optional success callback which will be run when the new conversation is successfully created
```

Reply:
How come we say "if" for failures and "when" for successes, is that deliberate?

Reply:
@ashfordneil not intentional. Now I'm overthinking this issue.